<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainw</class>
 <widget class="QMainWindow" name="mainw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainw</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="cb_prefocus">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Prefocus</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_expo">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Expo</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="num_disp_brightness">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>340</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>256</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Disp. brightness:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="num_pwr_mng">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>380</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>256</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Power managment:</string>
    </property>
   </widget>
   <widget class="QSlider" name="sld_battery">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>19</width>
      <height>251</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
    <property name="value">
     <number>3000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>381</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Menu</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="grf_display">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>258</width>
       <height>134</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_OK">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_startstop">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>start/stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pwr/mode</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_esc">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>320</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_power">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QDial" name="dial">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>161</width>
       <height>141</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="notchTarget">
      <double>10.000000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>300</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>esc</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_menu">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_3</zorder>
    <zorder>grf_display</zorder>
    <zorder>btn_startstop</zorder>
    <zorder>label</zorder>
    <zorder>btn_esc</zorder>
    <zorder>btn_power</zorder>
    <zorder>dial</zorder>
    <zorder>label_2</zorder>
    <zorder>btn_menu</zorder>
    <zorder>btn_OK</zorder>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Battery</string>
    </property>
   </widget>
   <widget class="QSlider" name="sld_light">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>19</width>
      <height>251</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
    <property name="value">
     <number>3000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Light</string>
    </property>
   </widget>
   <widget class="QSlider" name="sld_sound">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>19</width>
      <height>251</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
    <property name="value">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sound</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_adc_on">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC on</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="num_dbg">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>-1000000</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DBG:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_beepHi">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BeepHi</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_beepLo">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BeepLo</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
